x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_store_data_client.ts

```diff
@@ -278,41 +332,54 @@ export class EntityStoreDataClient {
   }
 
   public async start(entityType: EntityType, options?: { force: boolean }) {
+    const { namespace } = this.options;
     const descriptor = await this.engineClient.get(entityType);
     if (!options?.force && descriptor.status !== ENGINE_STATUS.STOPPED) {
       throw new Error(
-        `In namespace ${this.options.namespace}: Cannot start Entity engine for ${entityType} when current status is: ${descriptor.status}`
+        `In namespace ${namespace}: Cannot start Entity engine for ${entityType} when current status is: ${descriptor.status}`
```

Comment:
Unrelated: We should return the error message when the status is an error.

Reply:
But if we throw it just gets caught by any catch handlers in the requests, no?